source/languageHandler.py

```diff
@@ -362,7 +361,8 @@ def setLanguage(lang: str) -> None:
 	if trans is None:
 		trans = _createGettextTranslation("en")
 
-	trans.install(names=["pgettext", "npgettext"])
+	# #9207: Python 3.8 adds gettext.pgettext, so add it to the built-in namespace.
+	trans.install(names=["pgettext", "ngettext"])

```

Comment:
why was `npgettext` removed here? For 3.7 we didn't need to patch `ngettext`, only `npgettext`